README.md

```diff
@@ -3,22 +3,8 @@
 [![Lint](https://github.com/ForgeRock/iot-edge/workflows/golangci-lint/badge.svg)](https://github.com/ForgeRock/iot-edge/actions?query=workflow%3Agolangci-lint)
 [![go test](https://github.com/ForgeRock/iot-edge/workflows/go-test/badge.svg)](https://github.com/ForgeRock/iot-edge/actions?query=workflow%3Ago-test)
 
-# ForgeRock IoT Edge
+# ForgeRock Things
 
-ForgeRock IoT Edge is an open source project containing the IoT edge tier components of the ForgeRock Identity Platform.
+ForgeRock Things is an open source project containing the IoT edge tier components of the ForgeRock Identity Platform.
 
-<img src="docs/iot-edge-components.svg" width="450"/>
-
-## Thing SDK
-
-The _Thing SDK_ enables a _thing_, which can be either a physical _device_ or a software _service_, to register and
-authenticate without human interaction. Once registered, the _thing_ will be represented by a digital identity in the
-ForgeRock Identity Platform and can authenticate itself in order to interact with the platform tier.
-
-See the [getting started guide](docs/getting-started.md) for information about how to use the SDK.
-
-## Thing Gateway
-The _Thing Gateway_ is an application that enables more constrained devices to interact with the ForgeRock Identity
-Platform by acting as a proxy between a _thing_ and the Platform.
-
-See the [build the Gateway guide](docs/building-the-gateway.md) for information about building the Gateway.
+For more information about ForgeRock Things and how to use it, please refer to the [getting started guide](docs/getting-started.md). 
```

Comment:
This feels a bit light on text now. Since most people will automatically follow the link, should we move the getting started guide into this readme?

Reply:
The "getting started guide" will be replaced by the Backstage docs and the link will point to that once it is done.

If we find that there are repo specific information that does not fit in the Backstage doc then we can add it to this page.